meta/main.yml

```diff
@@ -1,6 +1,6 @@
 galaxy_info:
-  author: Irit Goihman
```

Comment:
not much has remained of her code, but let's keep her, and keep her description.

Reply:
Done